lib/client.js

```diff
@@ -44,7 +56,7 @@ class Client {
       Key: fileName,
       Body: body,
     };
-    await this.s3.upload(params).promise();
+    return await this.s3.upload(params).promise();
```

Comment:
Outside of try/catch blocks, `return await` is redundant. There's even an [ESLint rule](https://github.com/eslint/eslint/blob/master/docs/rules/no-return-await.md) to detect it, but it allows it in try/catch.